templates/load-message.lua

```diff
@@ -0,0 +1,57 @@
+-- Some parameters to control the behavior. Can be set via Redis keys.
+-- How long (in seconds) should a given buffer bucket be?
+local pBufferBucketInterval = tonumber(redis.call("GET", "conf-bufferBucketInterval")) or 60
+-- How many buffer buckets should we retain?
+local pBufferBucketCount = tonumber(redis.call("GET", "conf-bufferBucketCount")) or 20
+
+-- Derived parameters
+local pBufferLifetime = pBufferBucketInterval * pBufferBucketCount
+
+-- First, parse and restructure the incoming message.
+local message = cjson.decode(ARGV[1])
```

Comment:
Addressed this in https://github.com/aptible/gentlemanjerry/pull/21/commits/14b8e917ce85aa5006bd0ca67e2bc0fdd5537a5b